src/ui/mystery-encounter-ui-handler.ts

```diff
@@ -617,30 +619,32 @@ export default class MysteryEncounterUiHandler extends UiHandler {
 
       // Sets up the mask that hides the description text to give an illusion of scrolling
       const tooltipTextMaskRect = globalScene.make.graphics({});
+      const MaskHeight = 32;
       tooltipTextMaskRect.setScale(6);
       tooltipTextMaskRect.fillStyle(0xffffff);
       tooltipTextMaskRect.beginPath();
-      tooltipTextMaskRect.fillRect(this.tooltipContainer.x, this.tooltipContainer.y + 188.5, 150, 32);
+      tooltipTextMaskRect.fillRect(this.tooltipContainer.x, this.tooltipContainer.y + 188.5, 150, MaskHeight);
 
       const textMask = tooltipTextMaskRect.createGeometryMask();
       tooltipTextObject.setMask(textMask);
 
-      const tooltipLineCount = Math.floor(tooltipTextObject.displayHeight / 11.2);
+      const displayHeight = tooltipTextObject.displayHeight;
+      const scrollAmount = displayHeight - MaskHeight;
 
       if (this.tooltipScrollTween) {
         this.tooltipScrollTween.remove();
         this.tooltipScrollTween = undefined;
       }
 
       // Animates the tooltip text moving upwards
-      if (tooltipLineCount > 3) {
+      if (displayHeight > MaskHeight) {
```

Comment:
```suggestion
      if (displayHeight > maskHeight) {
```
That should be all of them. Otherwise lgtm!